package.json

```diff
@@ -99,7 +100,7 @@
     "@budibase/types": "0.0.0"
   },
   "engines": {
-    "node": ">=18.0.0 <19.0.0"
+    "node": ">=18.0.0 <21.0.0"
```

Comment:
We cannot remove the node18 engine until we update the `budibase-deploys` node versions